app/main.py

```diff
@@ -1,3 +1,27 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
```

Comment:
The `__add__` method should handle the case where `other` is not an instance of `Distance`. Consider adding an `else` clause to return `NotImplemented` if `other` is not a `Distance` instance.